[CI] DockerTests test022InstallPluginsFromLocalArchive failing

**Build Scans:**
- [elasticsearch-periodic-packaging #11428 / ubuntu-2404_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/2gcckzmcq6ezm)
- [elasticsearch-periodic-packaging #11428 / oraclelinux-9_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/46ry6qgmimpsg)
- [elasticsearch-periodic-packaging #11428 / rhel-9_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/ufuslwz2qezbs)
- [elasticsearch-periodic-packaging #11428 / debian-12_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/65ou32u3sbkze)
- [elasticsearch-periodic-packaging #11428 / rocky-9_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/f7kgxaucvmste)
- [elasticsearch-periodic-packaging #11428 / sles-15_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/7odnjljxxmsno)
- [elasticsearch-periodic-packaging #11428 / rocky-8_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/ztfr2m4nw4qyo)
- [elasticsearch-periodic-packaging #11428 / almalinux-8_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/tmqfy3hgwg4ik)
- [elasticsearch-periodic-packaging #11428 / debian-13_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/hw6vv2bn3glsw)
- [elasticsearch-periodic-packaging #11428 / rhel-8_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/pjcomyulr46ag)

**Reproduction Line:**
```
null
```

**Applicable branches:**
9.1

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.packaging.test.DockerTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test022InstallPluginsFromLocalArchive'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Expected analysis-icu available in archive
Expected: a collection with size <16>
     but: collection size was <17>
```

**Issue Reasons:**
- [9.1] 43 consecutive failures in test test022InstallPluginsFromLocalArchive
- [9.1] 2 consecutive failures in step ubuntu-2404_docker-cloud-ess_packaging-tests-unix
- [9.1] 2 consecutive failures in step ubuntu-2204_docker-cloud-ess_packaging-tests-unix
- [9.1] 3 consecutive failures in step almalinux-8_docker-cloud-ess_packaging-tests-unix
- [9.1] 2 consecutive failures in step debian-13_docker-cloud-ess_packaging-tests-unix
- [9.1] 2 consecutive failures in step oraclelinux-9_docker-cloud-ess_packaging-tests-unix
- [9.1] 2 consecutive failures in step rhel-9_docker-cloud-ess_packaging-tests-unix
- [9.1] 2 consecutive failures in step debian-12_docker-cloud-ess_packaging-tests-unix
- [9.1] 2 consecutive failures in step rocky-9_docker-cloud-ess_packaging-tests-unix
- [9.1] 2 consecutive failures in step sles-15_docker-cloud-ess_packaging-tests-unix
- [9.1] 2 consecutive failures in step almalinux-9_docker-cloud-ess_packaging-tests-unix
- [9.1] 2 consecutive failures in step opensuse-leap-15_docker-cloud-ess_packaging-tests-unix
- [9.1] 2 consecutive failures in step rhel-8_docker-cloud-ess_packaging-tests-unix
- [9.1] 2 consecutive failures in step rocky-8_docker-cloud-ess_packaging-tests-unix
- [9.1] 2 consecutive failures in step oraclelinux-8_docker-cloud-ess_packaging-tests-unix

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -337,9 +337,6 @@ tests:
 - class: org.elasticsearch.xpack.test.rest.XPackRestIT
   method: test {p0=ml/3rd_party_deployment/Test start deployment with low priority and max allocations more than one}
   issue: https://github.com/elastic/elasticsearch/issues/140610
-- class: org.elasticsearch.packaging.test.DockerTests
-  method: test022InstallPluginsFromLocalArchive
-  issue: https://github.com/elastic/elasticsearch/issues/141460
 
 # Examples:
 #

--- a/qa/packaging/src/test/java/org/elasticsearch/packaging/test/DockerTests.java
+++ b/qa/packaging/src/test/java/org/elasticsearch/packaging/test/DockerTests.java
@@ -213,7 +213,7 @@ public class DockerTests extends PackagingTestCase {
 
         listPluginArchive().forEach(System.out::println);
         assertThat("Expected " + plugin + " to not be installed", listPlugins(), not(hasItems(plugin)));
-        assertThat("Expected " + plugin + " available in archive", listPluginArchive(), hasSize(16));
+        assertThat("Expected " + plugin + " available in archive", listPluginArchive(), hasItems(containsString(plugin)));
 
         // Stuff the proxy settings with garbage, so any attempt to go out to the internet would fail
         sh.getEnv()

